src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
Binary compatibility break BTW. Requires a major version bump + review of dependent plugins